pylabnet/launchers/launch_gui.py

```diff
@@ -0,0 +1,65 @@
+""" Runs a GUI window
+
+Instantiates client to connect to Logger. Runs a Window server
+Continuously updates GUI configuration and output. Can press the stop button + close the window to deactivate the GUI
+Can connect to the GUI and update data by creating a GUI client.
+
+Must tie the GUI to a .ui file that can be created in QtDesigner
+"""
+
+from PyQt5 import QtWidgets
+
+from pylabnet.gui.pyqt.external_gui import Window, Service
+from pylabnet.core.generic_server import GenericServer
+from pylabnet.utils.logging.logger import LogClient
+
+import sys
+import socket
+
+
+def main():
+
+    # Retrieve GUI template from command line argument
+    gui_template = str(sys.argv[1])
+
+    # Instantiate logger
+    gui_logger = LogClient(
+        host='localhost',
+        port=12345,
+        module_tag='GUI module'
+    )
+
+    # Create app and instantiate main window
+    app = QtWidgets.QApplication(sys.argv)
+    main_window = Window(app, gui_template=gui_template)
+
+    # Instantiate GUI server
+    port_num = 12
```

Comment:
port number generation should also be intelligent. perhaps a util function for this?